plugins/module_utils/netbox_utils.py

```diff
@@ -306,10 +307,12 @@
     "nat_inside": set(["vrf", "address"]),
     "parent_region": set(["slug"]),
     "platform": set(["slug"]),
+    "port": set(["name"]),
```

Comment:
I don't think we need **port** or **protocol** here as they won't have to be searched.